apps/src/code-studio/progressRedux.js

```diff
@@ -611,7 +613,8 @@ const levelWithProgress = (
     status: status,
     isCurrentLevel: isCurrent,
     paired: levelPairing[level.activeId],
-    isLocked: locked
+    isLocked: locked,
+    teacherFeedbackReviewState: teacherFeedbackReviewState
```

Comment:
could be nice to add this to `LevelWithProgressType` in `progressTypes.js`

Reply:
Ah right, that's something that gets added in https://github.com/code-dot-org/code-dot-org/pull/41434 but I could add it here.

Reply:
Added!